src/CommunityToolkit.Maui.MediaElement/Extensions/VttParser.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Globalization;
+using System.Text.RegularExpressions;
+
+namespace CommunityToolkit.Maui.Extensions;
+
+static partial class VttParser
+{
+	static readonly string[] separator = ["\r\n", "\n"];
+	static readonly Regex timecodePattern = new((@"(\d{2}:\d{2}:\d{2}\.\d{3}) --> (\d{2}:\d{2}:\d{2}\.\d{3})"));
```

Comment:
looks like the same members as the other parser

Reply:
slight difference that affects the outcome. I had to double check.